packages/pn-commons/src/utils/costants.tsx

```diff
@@ -10,6 +10,8 @@ export const pagoPALink: {href: string; ariaLabel: string} = {
   ariaLabel: "Link: vai al sito di PagoPA S.p.A.",
 };
 
+export const URL_PRIVACY_LINK = 'https://notifichedigitali.it/cittadini/informativa-privacy'
```

Comment:
@maurizioflauti vorrei spezzare la prima parte (notifichedigitali.it) dal resto del path